src/applications/static-pages/cta-widget/components/messages/MultipleIds.jsx

```diff
@@ -16,9 +14,9 @@ const MultipleIds = () => {
         </p>
         <AdditionalInfo triggerText="Call the My HealtheVet help desk">
           <p>
-            Call us at <a href="tel:877-327-0022">877-327-0022</a>. We’re here
-            Monday through Friday, 8:00 a.m. to 8:00 p.m. ET. If you have
-            hearing loss, call TTY: <Telephone contact={CONTACTS.HELP_TTY} />.
+            Call us at <va-telephone contact="8773270022" />. We’re here Monday
+            through Friday, 8:00 a.m. to 8:00 p.m. ET. If you have hearing loss,
+            call <va-telephone contact={CONTACTS.HELP_TTY} tty />.
```

Comment:
Since the `CONTACTS` import is coming from the React component, this will need to be removed/updated as well. 

We have a directory of phone numbers on [the Storybook page](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default) as well if you need to reference which one is `CONTACTS.HELP_TTY`. There are other instances of this throughout that should be updated as well.

Reply:
The `CONTACTS` import at the top of the file needs to be removed completely and the prop `contact={CONTACTS.HELP_TTY}` should instead be`contact="8008778339"`

Reply:
the docs here seem to indicate that we're okay using CONTACTS.X as a property. (https://design.va.gov/storybook/?path=/docs/components-va-telephone--default)  Is that not accurate, and instead all contact numbers need to be converted to just being the actual number?

Reply:
Sorry for the confusion! The import to the React component needs to be updated: 

`import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';`

We will be removing the React component completely from the component library so this import will not longer function.

**Will No Longer Function**: `import { CONTACTS } from '@department-of-veterans-affairs/component-library/Telephone'`

Reply:
I'm honestly wrapping my head around this change right now too since the previously developer is no longer on the team but here is a link with maybe some more clarification about the import being different: https://github.com/department-of-veterans-affairs/component-library/pull/295#discussion_r809351674

And I think DST might want to update documentation to make this clearer as well (because even I was confused).

Reply:
Made these updates, thanks for pointing out!